community/transmission-daemon/build

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh -e
+
+./configure \
+    --prefix=/usr \
+    --sbindir=/usr/bin \
+    --mandir=/usr/share/man \
+    --sysconfdir=/etc \
+    --localstatedir=/var \
+    --disable-nls \
+    --enable-cli \
+    --enable-daemon \
+    --enable-utp \
+    --without-systemd-daemon \
+    --without-gtk
+
+make CXXFLAGS="$CXXFLAGS -std=c++11"
```

Comment:
`make -C daemon` perhaps?

Also, why the `-std=c++11`?

Reply:
> `make -C daemon` perhaps?
Needs to build libs first, so no.
> Also, why the `-std=c++11`?
Taken from another build, turns out unnecessary.

Reply:
> `make -C daemon` perhaps?

need to build libs first, so no

> Also, why the `-std=c++11`?

turns out unnecessary.